src/SFA.DAS.ApplyService.Application/Apply/IApplyRepository.cs

```diff
@@ -79,6 +79,8 @@ public interface IApplyRepository
 
         Task<bool> UpdateOversightReviewStatus(Guid applicationId, string oversightStatus, string userId, string userName);
         Task<ApplicationOversightDetails> GetOversightDetails(Guid applicationId);
+        Task<List<ApplicationOversightDownloadDetails>> GetOversightsForDownload(DateTime dateFrom, DateTime dateTo);
```

Comment:
just for consideration, potentially putting this on line 79 so it's next to the other ones